packages/telemetry/src/telemetryReporter.ts

```diff
@@ -173,7 +240,41 @@ export class SpawnedTelemetryReporter extends AsyncCreatable<TelemetryOptions> {
    */
   public sendTelemetryEvent(eventName: string, attributes: Attributes = {}): void {
     if (this.forkedProcess) {
-      this.forkedProcess.send({ eventName, attributes });
+      this.forkedProcess.send({ type: 'event', eventName, attributes });
+    }
+  }
+
+  /**
+   * Sends exceptopm to child process.
```

Comment:
```suggestion
   * Sends exception to child process.
```

Reply:
@brpowell Take a look at the changes in `fix: misspelled 'exception'`